src/meta/processors/schemaMan/Utils.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "meta/common/MetaCommon.h"
+
+#pragma once
```

Comment:
I  found another head file using #ifndef and #define

Reply:
Yes, I it's ok too. For example the headers of folly we used introduce `#pragma once` too.